QUICKSTART.md

```diff
@@ -0,0 +1,273 @@
+);
+tracerRegistry.addSpanProcessor(
+    BatchSpansProcessor.newBuilder(new LoggingExporter()).build()
+);
+tracerRegistry.addSpanProcessor(MultiSpanProcessor.create(Arrays.asList(
+            SimpleSpansProcessor.newBuilder(new LoggingExporter()).build(),
+            BatchSpansProcessor.newBuilder(new LoggingExporter()).build()
+)));
+```
+
+### Exporter
+
+Span Processor are initialized with an exporter which is responsible to send the telemetry data to your backend of choice.
+OpenTelemetry offers four exporters out of the box:
+- In Memory Exporter: keeps the data in memory, useful for debug.
+- Jaeger Exporter: prepare and send the collected telemetry data to a Jaeger backend via gRPC.
+- Logging Exporter: saves the telemetry data into log streams.
+- OpenTelemetry Exporter: sends the data to the [OpenTelemetry Collector] (not yet implemented).
+
+[OpenTelemetry Collector]: https://github.com/open-telemetry/opentelemetry-collector
+
+```java
+tracerRegistry.addSpanProcessor(SimpleSpansProcessor.newBuilder(
+    InMemorySpanExporter.create()
+).build());
+tracerRegistry.addSpanProcessor(SimpleSpansProcessor.newBuilder(
+    new LoggingExporter()
+).build());
+
+ManagedChannel jaegerChannel = ManagedChannelBuilder.forAddress([ip:String], [port:int]).usePlaintext().build();
+JaegerGrpcSpanExporter jaegerExporter = JaegerGrpcSpanExporter.newBuilder()
+    .setServiceName("example").setChannel(jaegerChannel).setDeadline(30000)
+    .build();
+tracerRegistry.addSpanProcessor(BatchSpansProcessor.newBuilder(
+    jaegerExporter
+).build());
+```
+
+# Tracing
+
+In the following, we present how to trace code using the OpenTelemetry API.
+**Note:** Methods of the OpenTelemetry SDK should never be called.
+ 
+First, the code must acquire a `tracer` which is responsible of create spans and interact with the in-process W3C trace-context.
+A tracer is acquired using the OpenTelemetry API specifying name and version parameters. 
+More information are available in [Obtaining a Tracer].
+
+```java
+Tracer tracer = OpenTelemetry.getTracerRegistry().get("instrumentation-library-name","semver:1.0.0");
+```
+
+[Obtaining a Tracer]: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/api-tracing.md#obtaining-a-tracer
+
+## Create basic Span
+To create a basic span, you only need to specify the name of the span. 
+Starting/Ending time and tracing-context are provided automatically by the OpenTelemetry API.
+```java
+Span span = tracer.spanBuilder("SpanName").startSpan();
+// your use case
+...
+span.end();
+```
+
+## Create nested Spans
+
+Most of the time, we want to correlate spans for nested operations. 
+OpenTelemetry supports distributed tracing within process and remote processes.
+For more details how to share context between remote processes, see [Context Propagation](#context-propagation).
+
+```java
+Span childLocalParent = tracer.spanBuilder("Child").setParent(parentSpan).startSpan();
```

Comment:
```suggestion
void a() {
  parentSpan = tracer.spanBuilder("a").startSpan();
  b();
  parentSpan.end();
}
void b() {
  Span childSpan = tracer.spanBuilder("b").setParent(parentSpan).startSpan();
  // do stuff
  childSpan.end();
}
```

And here it should be mentioned that one would usually "activate" the span using `Scope scope = tracer.withSpan(parentSpan)` to automatically propagate the parentSpan. Otherwise the parent would have to be kept somewhere accessible by or manually passed to `B`.